utils/python/CIME/buildnml.py

```diff
@@ -124,9 +126,9 @@ def build_data_nml(argv, compclass):
             # single-case restart for each instance
             rpointer = "rpointer." + compname
             if (os.path.isfile(os.path.join(rundir,rpointer)) and
-                (not os.path.isfile(os.path.join(rundir,rpointer+inst_string)))):
-                     shutil.copy(os.path.isfile(os.path.join(rundir,rpointer),
```

Comment:
The old code is broken. Someone was trying to copy the result of isfile, which is a bool.